.github/workflows/backend_unit_test.yml

```
@@ -0,0 +1,71 @@
+name: Backend Unit Test
+
+on:
+  pull_request:
+    paths:
+      - 'backend/pong/**'
+      - '.github/workflows/backend_unit_test.yml'
+
+jobs:
+  backend-unit-test:
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 10: did not find expected key

(syntax-check)